packages/eslint-plugin/src/worker.ts

```diff
@@ -32,7 +32,9 @@ async function actionSort(classes: string) {
 async function actionBlocklist(classes: string, id?: string) {
   const uno = await getGenerator()
   const extracted = await uno.applyExtractors(classes, id)
-  return [...extracted.values()].filter(i => uno.isBlocked(i))
+  const values = [...extracted.values()]
+  const blocked = await Promise.all(values.map(i => uno.isBlockedWithStrict(i)))
+  return values.filter((_, i) => blocked[i])
```

Comment:
If it's an eslint only feature, how about moving it out of core? Ideally we could have an option for it (I am ok with default strict)

Reply:
I prefer to support strict mode directly in `isBlocked`. I need to spend some time researching how to implement a simple variants of the sync API